backend/app/api/v1/endpoints/entities.py

```
@@ -84,29 +93,216 @@ async def _run_extraction_task(
+            except Exception:
+                ent = None
+        else:
+            ent = None
+
+        if ent is None:
+            extraction_succeeded = False
+            try:
+                async with entity_sem:
+                    ent = await extractor.extract_entities_async(text, i)
+                extraction_succeeded = True
+            except Exception as exc:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)